src/ukify/ukify.py

```diff
@@ -672,53 +683,64 @@ def call_systemd_measure(uki: UKI, opts: UkifyConfig, profile_start: int = 0) ->
                 continue
 
             # Check if this is a section we already covered above
-            if section.name in to_measure:
+            if section.name in unique_to_measure:
                 continue
 
-            to_measure[section.name] = section
+            unique_to_measure[section.name] = section
 
     if opts.measure:
-        pp_groups = opts.phase_path_groups or []
+        to_measure = dict((k, v) for k, v in unique_to_measure.items())
```

Comment:
Hmm, what is the point here? This makes a copy of the dictionary, but I don't understand why.

Reply:
The description of the measurement algorithm needs to be somewhere. What is measured and in what order.

Reply:
Technically there is no reason to make a copy and it can just be reused, but I wanted to clearly differentiate  `unique_to_measure` (which contains all the unique sections to be measured) and `to_measure` (temporary variable which contains all the section to measured on each iteration).

As for what is the measurement algorithm is, I have no idea. @poettering said that it would be enough to "measure" (that is, to call `systemd-measure` and concatenate outputs) all the combinations of .dtbauto sections with others, which is what I implemented here.